topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,297 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
 >
 {: .hands_on}
 
-Rather than looking at quality scores for each individual read, FastQC looks at quality collectively across all reads within a sample:
-
-![Per base sequence quality](../../images/quality-control/per_base_sequence_quality.png "Per base sequence quality")
-
-On the x-axis are the base position in the read, and on the y-axis quality scores. In this example, the sample contains reads that are 37 bp long. For each position, there is a box plotted to illustrate the distribution of values (with the whiskers indicating the 90th and 10th percentile scores). The blue line represents the mean score at the postion. The plot background is color-coded to identify good (green), acceptable (yellow), and bad (red) quality scores.
-
 > ### {% icon question %} Questions
 >
-> 1. Which Phred encoding is used in the FASTQ file for these sequences?
-> 2. How is the mean score changing along the sequence?
-> 3. Is this tendency seen in all sequences?
+> Which Phred encoding is used in the FASTQ file for these sequences?
 >
 > > ### {% icon solution %} Solution
-> > 1. The Phred scores are encoded using `Sanger / Illumina 1.9` (`Encoding` in the top table)
-> > 2. The mean score over the sequence is dropping at the end of the sequences. It is usual: the sequencers are incorporating more errored nucleotides at the end. But the score stays good: over 28.
-> > 3. The box plots are getting wider at the end of the sequences. It means a lot of sequences have their score dropping at the end of the sequence. At after 31 nucleotides, more than 10% of the sequences have scores below 20.
-> >
+> > The Phred scores are encoded using `Sanger / Illumina 1.9` (`Encoding` in the top table).
```

Comment:
1.8?

Reply:
From FastQC report, it is `Sanger / Illumina 1.9`

Reply:
Ah, OK. I guess (but not sure) 1.9 just has a different header format. IMHO the most important piece of information is if its phred33 or phred64 encoding. Everything else is just nasty details :)